celestia_app/upgrade_preloaded_test.go

```diff
@@ -39,19 +39,23 @@ func TestUpgradePreloaded(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error cloning instance: %v", err)
 	}
+	err = validator.AddVolume("/root/.celestia-app", "1Gi")
```

Comment:
I assume that the `root` user is bc the image `ghcr.io/celestiaorg/celestia-app:v0.12.2` which is not using the celestia user yet IIRC, isn't it?

Reply:
yes!